From 3b7cdce8c19fc1093e02c8c0cbbfbc456bea77fe Mon Sep 17 00:00:00 2001
In-Reply-To: <20210105130359.21139-1-avarab@gmail.com>
References: <20210105130359.21139-1-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 12 Jan 2021 00:45:06 +0100
Subject: [PATCH 08/22] mailmap tests: remove redundant entry in test
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove a redundant line in a test added in d20d654fe8 (Change current
mailmap usage to do matching on both name and email of
author/committer., 2009-02-08).

This didn't conceivably test anything useful and is most likely a
copy/paste error.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t4203-mailmap.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/t/t4203-mailmap.sh b/t/t4203-mailmap.sh
index dbd365681e..03a98d9635 100755
--- a/t/t4203-mailmap.sh
+++ b/t/t4203-mailmap.sh
@@ -414,7 +414,6 @@ test_expect_success 'Shortlog output (complex mapping)' '
 	Other Author <other@author.xx>   nick2 <bugs@company.xx>
 	Other Author <other@author.xx>         <nick2@company.xx>
 	Santa Claus <santa.claus@northpole.xx> <me@company.xx>
-	Santa Claus <santa.claus@northpole.xx> <me@company.xx>
 	EOF
 
 	echo three >>one &&
-- 
2.29.2.222.g5d2a92d10f8

